include/ovs_control.h

```diff
@@ -0,0 +1,143 @@
+// Copyright 2019 The Alcor Authors.
```

Comment:
looks like this is ovs code, let's update the license per team suggestion.

Reply:
// Copyright (c) 2008-2017, 2019 Nicira, Inc.
// Copyright 2019 The Alcor Authors - file modified.

two lines added.